code/modules/speech/modules/speech/modifiers/flock_gradient.dm

```diff
@@ -5,4 +5,48 @@
 	. = message
 
 	message.flags &= ~SAYFLAG_HAS_QUOTATION_MARKS
-	message.content = gradientText("#3cb5a3", "#124e43", "\"[message.content]\"")
+	message.content = "[MAKE_CONTENT_MUTABLE("\"")][message.content][MAKE_CONTENT_MUTABLE("\"")]"
+	APPLY_CALLBACK_TO_MESSAGE_CONTENT(message, CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(immutable_gradientText), "#3cb5a3", "#124e43"))
+
+
+/// A copy of `gradientText` that ensures that the HTML content is immutable.
+/proc/immutable_gradientText(message, color_1, color_2)
```

Comment:
Why not just a param?

Reply:
Because the `gradientText` arguments are in the following order: `color_1, color_2, message`; the regex expects `message` to be the first argument.

Reply:
Wrapper then? I am allergic to copy paste :P